scrapers/fireroad.py

```diff
@@ -26,34 +26,43 @@
 URL = "https://fireroad.mit.edu/courses/all?full=true"
 
 
-def parse_timeslot(day, slot):
-    """Parses a timeslot. Example: parse_timeslot("M", "10-11.30") -> [4, 3]
+def parse_timeslot(day, slot, pm):
```

Comment:
could you update the docstring at the top of the file?

Reply:
Oh, thanks for catching that. Fixed.